### ISSUE TITLE
Alias generator not working on inherited attributes

### ISSUE THREAD

**Issue by Patdue** [samuelcolvin/pydantic#903]
# Alias generator not working on inherited attributes
* OS: **Debian GNU/Linux 9 (stretch)**
* Python version `import sys; print(sys.version)`: **3.7.3**
* Pydantic version `import pydantic; print(pydantic.VERSION)`: **0.32.2**

On inherited attributes, the alias generators do not work, if the base class also has an alias generator set. Minimal example:

```py
from pydantic import BaseModel

def alias_generator_1(field_name: str) -> str:
    return field_name + '1'


def alias_generator_2(field_name: str) -> str:
    return field_name + '2'


class Base(BaseModel):
    x: int

    class Config:
        allow_population_by_alias = True
        alias_generator = alias_generator_1


class Impl(Base):
    y: int

    class Config(Base.Config):
        alias_generator = alias_generator_2


i = Impl(x=1, y=2)

# expected: {'x2': 1, 'y2': 2}
# actual: {'x1': 1, 'y2': 2}
print(i.dict(by_alias=True))
```

A simple fix is to just define the attribute again on the child like so:
```
class Impl(Base):
    x: int
    y: int

    class Config(Base.Config):
        alias_generator = alias_generator_2
```

But i would expect the `alias_generator` to be overridden for all attributes, when defined in subclasses.

**Comment by samuelcolvin** [comment]
thanks for reporting, agreed it should apply to all fields.

Could you try on master or v1.0b2 and confirm if it's still broken?

**Comment by Patdue** [comment]
Just tested it on `master` and can confirm it is still broken.

**Comment by samuelcolvin** [comment]
should be fixed in #904, could you try that and check it works for you.
